softwerkskammer/test/groupsAndMembers/groupsAndMembersService_test.js

```diff
@@ -179,6 +204,35 @@ describe('Groups and Members Service (getGroupAndMembersForList)', () => {
 
 });
 
+describe('Groups and Members Service (getOrganizersOfGroup)', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it('returns no organizer when there is no group', (done) => {
+    const groupId = 'not-existing-group';
+    thereIsNoGroupFor(groupId);
+    groupsAndMembersService.getOrganizersOfGroup(groupId, (error, organizers) => {
+      expect(organizers).to.be.empty();
+      done(error);
+    });
+  });
+
+  it('returns the organizer when there is one and the group exists', (done) => {
+    const groupId = 'group-with-one-organizer';
+    const organizerId = 'organizerId';
+    const organizer = new Member({id: organizerId});
+    const member = new Member({id: 'any-member-id'});
+    thereIsGroup(new Group({id: groupId, organizers: [organizerId]}));
+    groupHasMembers([organizer, member]);
+    groupsAndMembersService.getOrganizersOfGroup(groupId, (error, organizers) => {
+      expect(organizers).to.have.length(1);
+      expect(organizers[0].id()).to.equal(organizerId);
+      done(error);
+    });
+  });
+});
+
```

Comment:
Should we maybe have an additional test for the case that there are no organizers in a group? I can also add it later after pulling this request.

Reply:
done